templates/nextjs/sway-programs/script/src/main.sw

```diff
@@ -0,0 +1,5 @@
+script;
+
+fn main() -> bool {
+    return true;
+}
```

Comment:
Maybe this starter code could be better 🤔

Reply:
Yeah, this is the same as the simplest predicate. Perhaps something like this just to differentiate:
```rs
fn main(number_to_increment: u8) -> u8 {
    return number_to_increment + 1;
}
```